metrics/alertmanager/ticdc.rules.yml

```diff
@@ -1,6 +1,18 @@
 groups:
 - name: alert.rules
   rules:
+  - alert: cdc_multiple_owner
+    expr: sum(rate(ticdc_owner_ownership_counter[30s])) > 2
```

Comment:
why is `>2`

Reply:
There should not be two or more owners at the same time.

Reply:
Does `>2` mean at least three owners?

Reply:
Updated, thanks!